defaults/main.yml

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: MIT
+---
+# The provider for nbde_client role. Currently 'clevis' is supported.
+nbde_client_provider: clevis
+
+# nbde_client_bindings is a list, and its elements support the following keys:
+#
+# * device             - specifies the path of a LUKS device. (REQUIRED)
+# * pass               - specifies a valid passphrase for this device.
+# * keyfile            - specifies either the absolute or relative path of a
+#                        valid keyfile for the device pointed out by device.
+#
+#  * state             - specifies whether a binding with the configuration
+#                        described should be added or removed. Setting state
+#                        to present (the default) means a binding will be
+#                        added; setting state to absent means a binding will
+#                        be removed from the device/slot.
+#
+# * slot               - specifies the slot to use for the binding. Default
+#                        is 1.
+# * servers            - specifies a list of servers to bind to. To enable
+#                        high availability, simply specify more than one server
+#                        here.
+# * threshold          - specifies the threshold for the Shamir Secret Sharing
```

Comment:
Is there a default for this?  e.g. is the default value the number of servers in the `servers` list?

Reply:
The default is 1. You can add two servers and if at least one of them is available, it will be enough to perform the decryption. If you set it to 2, both of them will be required to be available for the decryption.